pkg/agent/config/config.go

```diff
@@ -285,6 +285,14 @@ func get(envInfo *cmds.Agent) (*config.Node, error) {
 		}
 	}
 
+	// If node-ip is set it will override flannel interface by default if not set
+	if envInfo.NodeIP != "" && len(envInfo.FlannelIface) == 0 {
```

Comment:
Since the IP is unique but the iface is not it is probably better to go the other way. So if NodeIP is not defined but FlannelIface is then set NodeIP. That way the same command can be used on multiple nodes. Thanks!

Reply:
Fixed, thanks!